srcpkgs/libevdev/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'libevdev'
 pkgname=libevdev
-version=1.9.1
+version=1.10.0
```

Comment:
Hm, this version (or the more recent 1.10.1) aren't in https://www.freedesktop.org/wiki/Software/libevdev/

Weird.

Reply:
Got upstream to fix :)